lib/led/rgb.js

```diff
@@ -405,4 +405,17 @@ RGB.prototype.stop = function() {
   return this;
 };
 
+RGB.prototype.intensity = function(intensity) {
+  var state = priv.get(this);
+  var currentIntensity = RGB.colors.reduce(function(current, color) {
+    return Math.max(state[color], current);
+  }, 0) / 255;
+  var scale = 1 / currentIntensity * Math.min(100, Math.max(0, intensity)) / 100;
+  var scaledColors = RGB.colors.reduce(function(current, color) {
+    return (current[color] = state[color] * scale, current);
+  }, {});
+
+  this.update(scaledColors);
```

Comment:
Yup. There's also the getter. I just wanted feedback on the approach (which turned out to generate lots of discussion), because I did the full implementation.